verifier.sh

```diff
@@ -0,0 +1,686 @@
+
+    dname="$1"
+    if [ "$GEM_ALWAYS_YES" != "true" -a -d "$dname" ]; then
+        echo "$dname already exists"
+        echo "press Enter to continue or CTRL+C to abort"
+        read a
+    fi
+    rm -rf $dname
+    mkdir -p $dname
+}
+
+#
+#  usage <exitcode> - show usage of the script
+#      <exitcode>    value of exitcode
+#
+usage () {
+    local ret
+
+    ret=$1
+
+    echo
+    echo "USAGE:"
+    echo "    $0 devtest <branch-name>"
+    echo "                                                 put oq-platform sources in a lxc,"
+    echo "                                                 setup environment and run development tests."
+    echo "    $0 prodtest <branch-name>"
+    echo "                                                 production installation and tests."
+    echo
+    exit $ret
+}
+
+#
+#  _wait_ssh <lxc_ip> - wait until the new lxc ssh daemon is ready
+#      <lxc_ip>    the IP address of lxc instance
+#
+_wait_ssh () {
+    local lxc_ip="$1"
+
+    for i in $(seq 1 20); do
+        if ssh $lxc_ip "echo begin"; then
+            break
+        fi
+        sleep 2
+    done
+    if [ $i -eq 20 ]; then
+        return 1
+    fi
+}
+
+#
+#  _devtest_innervm_run <branch_id> <lxc_ip> - part of source test performed on lxc
+#                     the following activities are performed:
+#                     - extracts dependencies from oq-{engine,hazardlib, ..} debian/control
+#                       files and install them
+#                     - builds oq-hazardlib speedups
+#                     - installs oq-engine sources on lxc
+#                     - set up postgres
+#                     - upgrade db
+#                     - runs celeryd
+#                     - runs tests
+#                     - runs coverage
+#                     - collects all tests output files from lxc
+#
+#      <branch_id>    name of the tested branch
+#      <lxc_ip>       the IP address of lxc instance
+#
+_devtest_innervm_run () {
+    local i old_ifs pkgs_list dep branch_id="$1" lxc_ip="$2"
+
+    trap 'local LASTERR="$?" ; trap ERR ; (exit $LASTERR) ; return' ERR
+
+    scp .gem_init.sh ${lxc_ip}:
+    scp .gem_ffox_init.sh ${lxc_ip}:
+
+    # build oq-hazardlib speedups and put in the right place
+    ssh -t  $lxc_ip "source .gem_init.sh"
+
+    ssh -t  $lxc_ip "rm -f ssh.log"
+
+    ssh -t  $lxc_ip "sudo apt-get update"
+    ssh -t  $lxc_ip "sudo apt-get -y upgrade"
+
+    ssh -t  $lxc_ip "sudo apt-get install -y build-essential python-dev python-imaging python-virtualenv git libxml2 libxml2-dev libxslt1-dev libxslt1.1 libblas-dev liblapack-dev curl wget xmlstarlet imagemagick gfortran python-nose libgeos-dev python-software-properties"
+    ssh -t  $lxc_ip "sudo add-apt-repository -y ppa:openquake-automatic-team/latest-master"
+    ssh -t  $lxc_ip "sudo apt-get update"
+    ssh -t  $lxc_ip "sudo apt-get install -y python-decorator python-h5py python-psutil python-concurrent.futures python-oq-hazardlib python-oq-engine"
```

Comment:
```bash
ssh -t  $lxc_ip "sudo apt-get install -y python-oq-hazardlib python-oq-engine"
```
is enough and then we don't need to update dependencies across several scripts if something changes in the deps